batch/batch/worker/worker.py

```diff
@@ -188,11 +189,13 @@ def compose(auth: Union[MutableMapping, str, bytes], registry_addr: str = None):
 log.info(f'INTERNET_INTERFACE {INTERNET_INTERFACE}')
 log.info(f'UNRESERVED_WORKER_DATA_DISK_SIZE_GB {UNRESERVED_WORKER_DATA_DISK_SIZE_GB}')
 log.info(f'ACCEPTABLE_QUERY_JAR_URL_PREFIX {ACCEPTABLE_QUERY_JAR_URL_PREFIX}')
+log.info(f'LOCATION {LOCATION}')
 
 instance_config = CLOUD_WORKER_API.instance_config_from_config_dict(INSTANCE_CONFIG)
 assert instance_config.cores == CORES
 assert instance_config.cloud == CLOUD
 
+REGION = instance_config.region_for(LOCATION)
```

Comment:
Can we just pass REGION as the env variable here instead?

Reply:
I thought I had that previously, but you suggested using the instance config. I could be wrong though. This question is related to the comment above about how to pass the ZONE.

Reply:
Ah sorry that's not what I meant. I meant that in your original commit I liked the gcp `create_instance.py` changes, with the caveat that you could get the region by using the `instance_config.region_for` method instead of doing a string split.

What I intended was that the environment variable given to the worker be `REGION` and that we get that value in the first line of the `create_vm_config` function by doing

```python
region = instance_config.region_for(location)
```